### PR TITLE
Added initial deliverable documents

### PR DESCRIPTION
**Pull request by AWhetter**

### CODE REVIEW

File: deliverables/common_problems.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+This page collates some problems that were hit when transitioning to Python 3
+and their respective solutions.
+
+## Boost in a Python 3 Flavour of CY2019
+
+Unfortunately Python 3.7.x cannot be added to the CY2019 version
+of the VFX Reference Platform as-is.
+Boost 1.66 fails to find Python correctly when building with Python support.
+The easiest fix we (DNEG) found was to upgrade Boost to 1.70,
+as this is the version of Boost defined in CY2020.
```
Comment (csyshing) [inline deliverables/common_problems.md:12]: Maybe not CY2019 only but for a CMake build system, we (Animal Logic) found that Pixar's USD has a very useful module to find the correct Boost Python version:
https://github.com/PixarAnimationStudios/USD/blob/master/cmake/defaults/Packages.cmake#L42-L86

File: deliverables/resources.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  This guide's biggest use is in outlining many things that changed between Python 2 and 3,
+  how to change code to adapt to those changes,
+  and even how common those changes come up in transitions.
+  The guide also gives a very basic strategy for transitioning a code base to Python 3.
+  I have found this guide most useful in preparing teaching material for developers.
+
+* A keynote by Instagram at PyCon (https://youtu.be/66XoCk79kjM?t=768):
+  The keynote describes Instagram's high level strategy for transitioning to Python 3.
+  This was very useful in coming up with our own strategy.
+  They also talk about some of the pitfalls they hit when transitioning.
```
Comment (csyshing) [inline deliverables/resources.md:20]: * Cheat Sheet for Python 2 and 3 compatible code (http://python-future.org/compatible_idioms.html)
  This is a sub page of Python `future` website. It provides examples of the differences,
  and comes with options we can consider when the code must be compatible for both Python 2 and 3.

Comment (AWhetter) [reply deliverables/resources.md:20]: I've added this entry.

File: deliverables/common_problems.md (hunk 1 above)
Comment (AWhetter) [reply deliverables/common_problems.md:12]: Please could you write this up and add it to the document after this PR is merged.

**Pull request merged**